nekoyume/Assets/_Scripts/UI/Widget/Popup/BattleResultPopup.cs

```diff
@@ -88,50 +88,35 @@ public struct DefeatTextArea
         private const int Timer = 10;
         private static readonly Vector3 VfxBattleWinOffset = new(-0.05f, 1.2f, 10f);
 
-        [SerializeField]
-        private CanvasGroup canvasGroup;
+        [SerializeField] private CanvasGroup canvasGroup;
```

Comment:
We need line change for any of attributes. Could you edit your convention settings?
```suggestion
        [SerializeField]
        private CanvasGroup canvasGroup;
```